SleepEngine/Engine/GameWindow.cpp

```diff
@@ -1,36 +1,24 @@
 #include "stdafx.h"
 #include "GameWindow.h"
-
+#include <Engine/EngineConfig.h>
+#include <Engine/GameTemplate/Game.h>
 
 BeginNamespaceSleep
 
 namespace
 {
-    float const FPS = 60.f;
-
     unsigned const GLFWVersionMajor = 3;
     unsigned const GLFWVersionMinor = 3;
 
     void framebufferSizeCallback(GLFWwindow* const window, int const width, int const height)
     {
         glViewport(0, 0, width, height);
-        GameWindow::instance().getCamera().setScreenSize(width, height);
+        Game::instance().getCamera().setScreenSize(width, height);
     }
 }
 
-GameWindow::GameWindow(size_t const width, size_t const height, std::string_view const title, Color const bgColor)
-    : m_bgColor(bgColor)
-    , m_camera(width, height)
-    , m_clock(FPS)
+GameWindow::GameWindow(size_t const width, size_t const height, std::string_view const title)
```

Comment:
[Optional] I guess we can stop using const arguments when passing by copy:)
```suggestion
GameWindow::GameWindow(size_t width, size_t height, std::string_view title)
```